test/e2e/plugin_lifecycle/plugin_lifecycle_helper_test.go

```diff
@@ -5,81 +5,39 @@
 package plugin
```

Comment:
This doesn't seem to be a test file, Generally in golang if the name of the file ends with `_test.go` it is considered a test file. Reference: https://ieftimov.com/posts/testing-in-go-naming-conventions/

This file is just a helper file. My suggestion is to rename this to `plugin_lifecycle_test_helper.go`. Also, there are many other places where we are using this type of naming. I would suggest we update the naming there as well.

Reply:
thank you, updated.